lecture-variables.html

```diff
@@ -119,11 +133,11 @@
 someNumber /= x
 </code></pre>
 
-
+(Optional) <b>Challenge</b>: what is the value of <code>someNumber</code> at the end of the above code snippet? Walk through each line of code and try to compute the value of <code>someNumber</code> yourself. Then, paste the snippet above into codepen to check your answer.
```

Comment:
A bigger problem with this is that `x` is undefined. So this code won't run. Do you want me to give `x` a value so this code is actually valid?